chain/miner.go

```diff
@@ -15,8 +15,8 @@ func (mp *MessageProducer) MinerConstructor(to, from address.Address, params pow
 	ser := MustSerialize(&params)
 	return mp.Build(to, from, builtin_spec.MethodsMiner.Constructor, ser, opts...)
 }
-func (mp *MessageProducer) MinerControlAddresses(to, from address.Address, params *adt.EmptyValue, opts ...MsgOpt) *types.Message {
-	ser := MustSerialize(params)
+func (mp *MessageProducer) MinerControlAddresses(to, from address.Address, params adt.EmptyValue, opts ...MsgOpt) *types.Message {
```

Comment:
IMO this is a regression. _All_ the params objects should be passed by pointer, we just didn't fix the rest yet.

Reply:
Alright. As a final commit to this PR I will regenerate all these and with params passes by reference.